lib/simplifyExpression/basicsSearch/removeAdditionOfZero.js

```diff
@@ -1,30 +1,30 @@
-const clone = require('../../util/clone');
+const {rewriteNode} = require('math-rules');
 
 const ChangeTypes = require('../../ChangeTypes');
+const defineRuleString = require('../../util/defineRuleString');
 const Node = require('../../node');
 
+// e.g. 2 + 0 -> 2
+const REMOVE_ADDING_ZERO = defineRuleString('#a + 0', '#a');
+
+// e.g. 0 + 2 -> 2
+const REMOVE_ADDING_ZERO_REVERSE = defineRuleString('0 + #a', '#a');
+
```

Comment:
I haven't had time to document `math-rules` outside of some comments in the codebase itself.  It would be good if there were docs so that people using it in `mathsteps` have something to refer to when working with it.